index.html

```diff
@@ -91,16 +109,17 @@
         <!-- Sponsor links -->
         <br>
         <h3>Sponsored by:</h3>
-        <p><a href="https://www.browserstack.com/" target="_blank"><img src="img/browserstack_logo.png" alt="Browserstack logo"></a></p>
+        <p>
+            <a href="https://www.browserstack.com/" target="_blank"><img src="img/browserstack_logo.png" alt="Browserstack Sponsored Image"></a>
```

Comment:
`"Browserstack logo" was fine, I don't think `"Browserstack Sponsored Image"` is necessary

Same goes for TrackJS